$NetBSD: patch-Makefile,v 1.4 2024/10/13 21:04:52 yhardy Exp $

Let pkgsrc add the shebang to the fennel launcher script
Install manual pages relative to PKGMANDIR

--- Makefile.orig	2024-08-25 01:34:54.000000000 +0000
+++ Makefile
@@ -46,7 +46,7 @@ format: ; for f in $(SRC); do fnlfmt --f
 
 # All-in-one pure-lua script:
 fennel: src/launcher.fnl $(SRC) bootstrap/view.lua
-	@echo "#!/usr/bin/env $(LUA)" > $@
+	@echo "#!$(LUA_INTERPRETER)" > $@
 	@echo "-- SPDX-License-Identifier: MIT" >> $@
 	@echo "-- SPDX-FileCopyrightText: Calvin Rose and contributors" >> $@
 	FENNEL_PATH=src/?.fnl $(LUA) bootstrap/aot.lua $< --require-as-include >> $@
@@ -125,8 +125,8 @@ $(BIN_LUA_DIR)/src/liblua-mingw.a: $(BIN
 
 ## Install-related tasks:
 
-MAN_DOCS := man/man1/fennel.1 man/man3/fennel-api.3 man/man5/fennel-reference.5\
-	    man/man7/fennel-tutorial.7
+MAN_DOCS := man1/fennel.1 man3/fennel-api.3 man5/fennel-reference.5\
+	    man7/fennel-tutorial.7
 
 # The empty line in maninst is necessary for it to emit distinct commands
 define maninst =
@@ -138,7 +138,7 @@ install: fennel fennel.lua
 	mkdir -p $(DESTDIR)$(BIN_DIR) && cp fennel $(DESTDIR)$(BIN_DIR)/
 	mkdir -p $(DESTDIR)$(LUA_LIB_DIR) && cp fennel.lua $(DESTDIR)$(LUA_LIB_DIR)/
 	$(foreach doc,$(MAN_DOCS),\
-		$(call maninst,$(doc),$(DESTDIR)$(MAN_DIR)/$(doc)))
+		$(call maninst,man/$(doc),$(DESTDIR)$(MAN_DIR)/$(doc)))
 
 uninstall:
 	rm $(DESTDIR)$(BIN_DIR)/fennel
